wicketstuff-restannotations-parent/restannotations/src/main/java/org/wicketstuff/rest/resource/AbstractRestResource.java

```diff
@@ -833,4 +857,13 @@ public MethodMappingInfo getMethodInfo(Method method)
 	{
 		return mappedMethodsInfo.get(method);
 	}
+
+	/**
+	 * Gets the Supplier used to retrieve the Locale
+	 * @return the Supplier used to retrieve the Locale
+	 */
+	public Supplier<Locale> getLocaleSupplier()
```

Comment:
What's a purpose of this method?

Reply:
It's just a dumb getter. Do you think it should left out?

Reply:
I can't image why somebody would need this to get it outside of the class itself. Otherwise I think it should not harm.